examples/net_framework/CSharpExamples/DeepSpeechClient/DeepSpeech.cs

```diff
@@ -182,7 +184,14 @@ public unsafe void DiscardStream()
         /// <returns>The STT result. The user is responsible for freeing the string.  Returns NULL on error.</returns>
         public unsafe string SpeechToText(short[] aBuffer, uint aBufferSize, uint aSampleRate)
         {
-            return NativeImp.DS_SpeechToText(_modelStatePP, aBuffer, aBufferSize, aSampleRate);
+            var res = NativeImp.DS_SpeechToText(_modelStatePP, aBuffer, aBufferSize, aSampleRate);
+
+            int len = 0;
+            while (Marshal.ReadByte(res, len) != 0) ++len;
+            byte[] buffer = new byte[len];
+            Marshal.Copy(res, buffer, 0, buffer.Length);
+            return Encoding.UTF8.GetString(buffer);
```

Comment:
Can these lines instead just be `return Marshall.PtrToStringUTF8(res)`?

Reply:
Seems no, there's no direct Marshal for UTF-8.

Reply:
Ah, looks like it was added in a version of .NET that's more recent than the one we use: https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.marshal.ptrtostringutf8